client/src/rpc_response.rs

```diff
@@ -305,7 +305,7 @@ pub struct RpcBlockProduction {
     pub range: RpcBlockProductionRange,
 }
 
-#[derive(Serialize, Deserialize, Clone)]
+#[derive(Serialize, Deserialize, Clone, PartialEq, Eq)]
```

Comment:
Can you add `Debug` here as well?

Reply:
There's already a custom impl on line 317